### DISCUSSION TITLE
Drag and Drop Pages into other Spaces

### DISCUSSION THREAD

**Discussion by florianbachl** [GitbookIO/community#736] [Feature Requests]
I am using Github more as a personal knowledge and note base. So I have a space called "Recent Topics", where I gather info on a topic that comes up. When I am done, I would love to move the Page to a different Space. This is currently not possible as far as I have tried.

An example would be when we are having an incident and I am gathering info on everything in my "Recent Topics". After the incident, I want to transform the document into a post mortem exposition doc. This of course needs to move to a different space. 

Not having this feature means that I have to copy paste the whole text manually.

**Comment by kfc2023** [comment]
I have multiple pages I need to move to a different space, and only one repo available for my content. That means copy/paste, copy/paste, copy/paste...
